example/w-live-reload/README.md

```diff
@@ -2,83 +2,20 @@
 
 <br>
 
-This example shows a simple live reloading setup. It works by injecting a script
-into the `<head>` of HTML documents. The script opens a WebSocket back to the
-server. If the WebSocket gets closed, the script tries to reconnect. When the
-server comes back up, the client is able to reconnect, and reloads itself.
+This example shows a simple live reloading setup using the `Dream.livereload`
+middleware. It works by injecting a script into the `<head>` of HTML documents.
+The script opens a WebSocket back to the server. If the WebSocket gets closed,
+the script tries to reconnect. When the server comes back up, the client is able
+to reconnect and reloads itself.
```

Comment:
This example should show how to disable the middleware conditionally for production.